client/payment-details/summary/index.tsx

```diff
@@ -367,18 +413,76 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 						<CardFooter className="payment-details-capture-notice">
 							<div className="payment-details-capture-notice__section">
 								<div className="payment-details-capture-notice__text">
-									{ `${ __(
-										'You need to capture this charge before',
-										'woocommerce-payments'
-									) } ` }
-									<b>
-										{ dateI18n(
+									{ createInterpolateElement(
+										__(
+											'You need to <a>capture</a> this charge in ',
+											'woocommerce-payments'
+										),
+										{
+											a: (
+												// eslint-disable-next-line jsx-a11y/anchor-has-content, react/jsx-no-target-blank
+												<a
+													href="https://woocommerce.com/document/woocommerce-payments/settings-guide/authorize-and-capture/#capturing-authorized-orders"
+													target="_blank"
+													rel="noreferer"
+												/>
+											),
+										}
+									) }
+									<abbr
+										title={ dateI18n(
 											'M j, Y / g:iA',
-											moment
-												.utc( authorization.created )
-												.add( 7, 'days' )
+											new Date(
+												Date.UTC(
+													new Date(
+														authorization.created
+													).getFullYear(),
+													new Date(
+														authorization.created
+													).getMonth(),
+													new Date(
+														authorization.created
+													).getDate() + 7,
+													new Date(
+														authorization.created
+													).getHours(),
+													new Date(
+														authorization.created
+													).getMinutes(),
+													new Date(
+														authorization.created
+													).getSeconds()
+												)
+											)
```

Comment:
We might be missing the `timezone` argument in the call to the dateI18n function. 
```suggestion
											),
											'UTC'
```
Can you give it a try and let me know if that improves things?

Reply:
I have added it just now 👍 . Thanks